vault-consul-acl/step4.md

```diff
@@ -8,7 +8,7 @@ You can verify the Consul server started correctly by checking the logs.
 `export CONSUL_SERVER_TOKEN=$(cat consul.server  | grep token  | awk '{print $2}')`{{execute T1}}
 
 
-Finally you can apply the token to the server as its `agent` token.
+Finally, you can apply the token to the server as its `agent` token.
```

Comment:
```suggestion
Finally, you can apply the token to the server as its `agent` token.


<!-- Suggestion
I think it would be good to note in this step the relation between accessorId/secretID/token for Consul and Vault. Do they map to each other? Also we should note that the secretID for Consul is how you refer to the token on the cli/api and in files.
-->

```